python/lsst/fgcmcal/fgcmMakeLut.py

```diff
@@ -388,15 +398,27 @@ def runQuantum(self, butlerQC, inputRefs, outputRefs):
                 (filterHandle.dataId["physical_filter"], filterHandle.dataId["detector"]): filterHandle
                 for filterHandle in filterHandles
             }
+            filterToBand = {
```

Comment:
Was there supposed to be an alternative in the `else` block, or did you just forget to remove this? The lines immediately after/outside the else block look identical.